app/models/grda_warehouse/cohorts/cohort_column_title.rb

```diff
@@ -0,0 +1,32 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+# This model is an intermediate model used to hold data generated from existing cohorts.
+# The table is maintained by a script and is in the shape we expect to exist after the
+# next cohort rewrite.
+module GrdaWarehouse::Cohorts
+  class CohortColumnTitle < GrdaWarehouseBase
```

Comment:
I probably am missing something but is 'title' the right name for this model? The properties don't align with with what I would expect to find in a title, (title.title, title.data_type).